Jenkinsfile

```diff
@@ -72,15 +72,15 @@ pipeline {
                     virtualenv venv
                 fi
                 . venv/bin/activate
+                export PIP_USER=false
 
-                python -m pip install -U pip
-                pip3 install pypandoc==1.7.5
+                pip3 install pypandoc
 
                 # Remove pydoop from requirements before it's installed.
                 awk '!/pydoop.*/' requirements.txt > temp && mv temp requirements.txt
 
                 pip3 install -r requirements.txt
-                pip3 install pyspark==2.4.0
+                pip3 install pyspark==3.2.3 or pip3 install pyspark==3.3.0
```

Comment:
I wasn't quite sure if we should literally put in the "or pip3 install pyspark==3.3.0" or not.

Reply:
Yeah, it looks like it should be one or the other???